src/mlpack/methods/ann/ffn_impl.hpp

```diff
@@ -496,6 +496,43 @@ void FFN<OutputLayerType, InitializationRuleType,
       network[network.size() - 1]);
 }
 
+template<typename OutputLayerType, typename InitializationRuleType,
+         typename... CustomLayers>
+void FFN<OutputLayerType, InitializationRuleType, CustomLayers...>::summary(
+    arma::mat& inputs)
+{
+  // Do a Forward pass to initialize outputParameter in each layer.
+  if (parameter.is_empty())
+    ResetParameters();
+  if (!deterministic)
+  {
+    deterministic = true;
+    ResetDeterministic();
+  }
+  Forward(std::move(inputs));
+
+  long long totalParams = 0;
+
+  std::cout << std::setfill('-') << std::setw(60) << '\n';
```

Comment:
Instead of using `std::cout` use `Log::Info` .

Reply:
Oh right. Thanks for pointing that out.